src/modules/material-requests/services/material-requests.service.ts

```diff
@@ -90,6 +124,6 @@ export class MaterialRequestsService {
 			throw new NotFoundException(`Material Request with id '${id}' not found`);
 		}
 
-		return this.adapt(materialRequestResponse.app_material_requests[0]);
+		return this.adapt(materialRequestResponse.app_material_requests[0] as GqlMaterialRequest);
```

Comment:
strange that you need to cast this type. Typescript should be able to figure this out 🤔